index.js

```diff
@@ -991,3 +1023,20 @@ function updateProposalsCount (count) {
   var numberField = document.querySelector('#proposals-count-number')
   numberField.innerText = (count.toString() + ' proposal' + (count !== 1 ? 's' : ''))
 }
+
+function updatedFilterStatus () {
```

Comment:
I think this should be named `updateFilterStatus`, right? (Rather than update**d**.)

Reply:
Yes, exactly